doc/api/stream.md

```diff
@@ -434,13 +439,16 @@ See also: [`writable.cork()`][].
 <!-- YAML
 added: v0.9.4
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/11608
+    description: The `chunk` argument can now be a `Uint8Array` instance.
   - version: v6.0.0
     pr-url: https://github.com/nodejs/node/pull/6170
     description: Passing `null` as the `chunk` parameter will always be
                  considered invalid now, even in object mode.
 -->
 
-* `chunk` {String|Buffer} The data to write
+* `chunk` {String|Buffer|Uint8Array} The data to write
```

Comment:
This doesn't match the `chunk` description of `.end()` which is confusing, specifically the 'any' part.